chrysom/store.go

```diff
@@ -54,9 +54,9 @@ func (listener ListenerFunc) Update(items []model.Item) {
 
 type Reader interface {
 	// GeItems will return all the current items or an error.
-	GetItems(owner string) ([]model.Item, error)
+	GetItems(*GetItemsInput) (*GetItemsOutput, error)
 
-	Start(ctx context.Context) error
+	Start(ctx context.Context, input *GetItemsInput) error
```

Comment:
This no longer matches the uber.fx hook struct for starting up an application. 
https://pkg.go.dev/go.uber.org/fx#Hook

Reply:
Ah, good catch 👍